src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -152,11 +155,11 @@ public static int parseReviewRating(String reviewRating) throws ParseException {
     }
 
     /**
-     * Parses {@code reviewDate} to String value corresponding to date.
+     * Parses {@code reviewDate} to corresponding date.
      */
-    public static String parseReviewDate(String reviewDate) {
+    public static Date parseReviewDate(String reviewDate) throws java.text.ParseException {
         requireNonNull(reviewDate);
-        String trimmedDate = reviewDate.trim();
-        return trimmedDate;
+        DateFormat df = new SimpleDateFormat("dd/MM/yyyy");
+        return df.parse(reviewDate);
```

Comment:
i think you should keep the trimming of the date! then you just parse the trimmed date LMAO idk sorry i'm just tryna thing of cases where the code could fail